### PR TITLE
[ON HOLD] Changelings: The Changelinging

### PR DESCRIPTION
**Pull request by ghost**
_**ON HOLD DUE TO ~FEATURE~ DEEP FREEZE, REOPEN AFTER FREEZE IS OVER**_

:cl: More Robust Than You
balance: The abilities of changelings have been COMPLETELY overhauled!
/:cl:

I ditched the old Evolution points system for a new one! You start out with 4 evo points... enough for fleshmend and tiring sting or tiring sting and mutesting... You gain more by absorbing people. One absorption = 2 additional evo points. You can't readapt anymore.

Tiring Sting has been added, which is kinda like the Peacekeeper borg's hypospray... but it has some morphine, too.

Also, lings can't revive or fleshmend if on fire.

Hivemind chat is no longer :g/.g and is it's own verb now. Hivemind-Say is given to lings and linked humans.

### CODE REVIEW

**Comment by ShizCalev** [comment]
Freeze on antag stuff, as well as balance changes #28223

File: code/game/gamemodes/changeling/powers/tiny_prick.dm [hunk 1]
```diff
@@ -183,29 +131,30 @@
...
-	to_chat(target, "<span class='danger'>Your eyes burn horrifically!</span>")
-	target.become_nearsighted()
-	target.blind_eyes(20)
-	target.blur_eyes(40)
+/obj/effect/proc_holder/changeling/sting/para/sting_action(mob/user, mob/living/carbon/target)
+	add_logs(user, target, "stung", "para sting")
+	to_chat(target, "<span class='userdanger'>Your muscles ache and feel weak, like jelly... can't stand..... Aghh!!</span>")
+	target.emote("cough")
+	target.Knockdown(300)
+	target.Stun(300)
```
Comment (ChangelingRain) [inline code/game/gamemodes/changeling/powers/tiny_prick.dm]: you only need Knockdown here; Stun is redundant if you already have Knockdown.

**Comment by Repukan** [comment]
Parasting is the worst ability that has ever been added, and readding it under the pretense of a nerf just makes it thrash.

**Comment by Anonmare** [comment]
Couldn't we replace parasting with the Peacekeeper's tiring solution? It deals stamina damage and  caps out at doing 50 - enough to cripple a target's movement speed and keep them from running

**Comment by MoreRobustThanYou** [comment]
Good idea!

**Comment by MoreRobustThanYou** [comment]
Ok, parasting has been replaced with Tire Sting (still called para in the code, but who cares). It injects peacekeeper dizzy/tiring solution, and a bit of morphine. It's not insta stun, but it will sedate your target given time...

**Comment by KorPhaeron** [comment]
https://github.com/tgstation/tgstation/pull/28223

Ongoing freeze, can consider this when it is over

**Comment by Ninja137** [comment]
If you're going to try and do something like this, then you really need to include some sort of end-game thing for them to actually be going for. As-is, even a fully-evoed ling is... just kind of shit. You don't have enough chemical regen/capacity to maintain any big fights, stuns still rape you to death and you can't Adrenal enough to negate them, Augmented Vision is still bugged and gives X-ray  instead of Thermal.

**Comment by MoreRobustThanYou** [comment]
Maybe I can add horrorform lings like on vg/yogs/goon that requires like, 14 evo points to evolve (5 ABSORBS if they didn't spend their starting points)

**Comment by MoreRobustThanYou** [comment]
currently redoing hivemind chat

**Comment by MoreRobustThanYou** [comment]
Hivemind refactor has been pushed... won't update PR until it can be opened again

**Comment by Anonmare** [comment]
Oh I just remembered, morphine in low amount is a speed-enhancing chem.

**Comment by MoreRobustThanYou** [comment]
the Tiring sting also includes Tiring and Dizzying solution (peacekeepers use this)

**Comment by PKPenguin321** [comment]
I hate it

A rework might be cool but not this

**Comment by MoreRobustThanYou** [comment]
@PKPenguin321 would you say it's better than current lings?

**Comment by PKPenguin321** [comment]
No, this restricts them way too much. Current ling at least has the potential to be cool even if it's not often used that way

**Comment by Supermichael777** [comment]
Yeah their powers are set up to blast a single persons ass. outside of that their trash

**Comment by MoreRobustThanYou** [comment]
this PR makes changelings mostly DEFENSIVE at the round start (they start with 4 DNA points, and already have LR/monkey form/biodegrade

**Comment by Ninja137** [comment]
Yeah, still doesn't address the problem that lings don't have anything that they're going for, despite having a pseudo-progression system. If you're gonna give them point-increases, then they probably need some form of scaling, or at least some sort of end-game type thing.

**Comment by MoreRobustThanYou** [comment]
@KorPhaeron could you reopen this?

**Comment by KorPhaeron** [comment]
No because then it will open at the back of the PR list and I don't want to reopen 40 PRs

Responsibility of people who open things during the freeze to reopen them as a new PR

**Comment by MoreRobustThanYou** [comment]
k

**Pull request closed without merging**